src/utils/cn.ts

```diff
@@ -0,0 +1,3 @@
+import cn from 'bem-cn-lite';
+
+export {cn};
```

Comment:
Make file with `{cn}` reexport due to two reasons:
1. Faster to import, since it named export, while lib export is default
2. Easier to update, if lib migration will be needed

Didn't update every usage, but will use it at least in new components.